cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -331,16 +330,20 @@ public <T extends GenerableArtifact> void generate(Path inputFile, Object input,
         inputFile);
     report = cg.generate(input, generableArtifacts, this.outputRootPath.toAbsolutePath(), false, (task, progress) -> {
     });
-    ValidationUtils.checkGenerationReport(report);
-    Set<Path> generatedJavaFiles = report.getGeneratedFiles().stream().filter(e -> e.getFileName().endsWith(".java"))
-        .collect(Collectors.toSet());
-    if (!generatedJavaFiles.isEmpty()) {
-      try {
-        ParsingUtils.formatJavaSources(generatedJavaFiles);
-      } catch (FormatterException e) {
-        LOG.warn(
-            "Generation was successful but we were not able to format your code. Maybe you will see strange formatting.");
+    boolean generationSuccess = ValidationUtils.checkGenerationReport(report);
+    if (generationSuccess) {
+      Set<Path> generatedJavaFiles = report.getGeneratedFiles().stream().filter(e -> e.getFileName().endsWith(".java"))
+              .collect(Collectors.toSet());
+      if (!generatedJavaFiles.isEmpty()) {
+        try {
+          ParsingUtils.formatJavaSources(generatedJavaFiles);
+        } catch (FormatterException e) {
+          LOG.warn(
+                  "Generation was successful but we were not able to format your code. Maybe you will see strange formatting.");
+        }
       }
+    } else {
+      LOG.error("Generation not successful! For more info execute in verbose mode (-v).");
```

Comment:
Should be moved to a constant, as this message is being used twice already.

Reply:
Moved the verbose mode hint into a MessageConstant.